README.md

```diff
@@ -292,42 +292,46 @@ import 'highlight.js/styles/github.css';
 
 ### Browser ES6 Modules
 
-*Note: You need to install `@highlightjs/cdn-assets` package instead of `highlight.js`. See [Download prebuilt CDN assets](#download-prebuilt-cdn-assets)*
+*Note: For now you'll want to install `@highlightjs/cdn-assets` package instead of `highlight.js`.
+See [Download prebuilt CDN assets](#download-prebuilt-cdn-assets)*
 
 To import the library and register only those languages that you need:
 
 ```js
-import hljs from './node_modules/@highlightjs/cdn-assets/es/core.js';
-import javascript from './node_modules/@highlightjs/cdn-assets/es/languages/javascript.min.js';
+import hljs from './assets/js/@highlightjs/cdn-assets/es/core.js';
+import javascript from './assets/js/@highlightjs/cdn-assets/es/languages/javascript.min.js';
 
 hljs.registerLanguage('javascript', javascript);
 ```
 
 To import the library and register all languages:
 
 ```js
-import hljs from './node_modules/@highlightjs/cdn-assets/es/highlight.js';
+import hljs from './assets/js/@highlightjs/cdn-assets/es/highlight.js';
 ```
 
-*Note: The above code works only when the file containing the JavaScript code and the `node_modules` folder are inside same folder. If that is not the case, you have to modify the path in import statement accordingly.*
+*Note: The path to these files will depend on how your individiaul project is configured,
```

Comment:
In addition to this, I think it is better to be more explicit by including instruction to copy `@highlightjs/cdn-assets` from node_modules (or CDN) into `assets/js/@highlightjs/cdn-assets`.